src/coprocessor/codec/mysql/time.rs

```diff
@@ -387,6 +577,163 @@ impl Time {
             Ok(())
         }
     }
+
+    fn convert_date_format(&self, b: char) -> Result<String> {
+        match b {
+            'b' => {
+                let m = self.time.month();
+                if m == 0 || m > 12 {
+                    Err(box_err!("invalid time format"))
+                } else {
+                    Ok(MONTH_NAMES_ABBR[(m - 1) as usize].to_string())
```

Comment:
Could we use `MONTH_NAMES` like `tidb` did?

Reply:
I think the code here is more easier to understand, BTW performance is better.